mock-core-configs/etc/mock/templates/centos-7.tpl

```diff
@@ -27,74 +31,113 @@ best=1
 protected_packages=
 user_agent={{ user_agent }}
 
-{% set centos_7_gpg_keys = 'file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-7' %}
+{% set centos_7_arch_gpg_key -%}
 {% if target_arch in ['ppc64le', 'ppc64'] %}
-{%   set centos_7_gpg_keys = centos_7_gpg_keys + ',file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-SIG-AltArch-7-' + target_arch %}
-{% elif target_arch in ['aarch64'] %}
-{%   set centos_7_gpg_keys = centos_7_gpg_keys + ',file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-7-aarch64' %}
-{% endif %}
+       file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-SIG-AltArch-7-{{ target_arch }}
+{%- elif target_arch == 'aarch64' %}
+       file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-7-aarch64
+{%- endif -%}
+{% endset -%}
 
 # repos
 [base]
 name=CentOS-$releasever - Base
-mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
-failovermethod=priority
-gpgkey={{ centos_7_gpg_keys }}
+mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
+#baseurl=http://mirror.centos.org/$contentdir/$releasever/os/$basearch/
 gpgcheck=1
-skip_if_unavailable=False
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-7
+       {{- centos_7_arch_gpg_key }}
 
 [updates]
 name=CentOS-$releasever - Updates
-enabled=1
-mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
-failovermethod=priority
-gpgkey={{ centos_7_gpg_keys }}
+mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
+#baseurl=http://mirror.centos.org/$contentdir/$releasever/updates/$basearch/
 gpgcheck=1
-skip_if_unavailable=False
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-7
+       {{- centos_7_arch_gpg_key }}
 
 [extras]
 name=CentOS-$releasever - Extras
-mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=extras
-#baseurl=http://mirror.centos.org/centos/$releasever/extras/$basearch/os/
-failovermethod=priority
```

Comment:
failovermethod=priority should stay enabled in EL7 configs

Reply:
Ok, should we use it in EL8 as well? Because it is missing from all repos there (except one).